UI/Task_2/test_verify_adding_item_to_Wishlist.py

```diff
@@ -0,0 +1,28 @@
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.webdriver.support.ui import WebDriverWait
+
+from configuration import driver
+
+def test_add_item_to_wishlist(driver):
+    driver.get('https://demowebshop.tricentis.com/50s-rockabilly-polka-dot-top-jr-plus-size')
+
+    WebDriverWait(driver, 10).until(
+        EC.presence_of_all_elements_located((By.CSS_SELECTOR, '.item-box'))
+    )
```

Comment:
You can move it to separated method\fixture and reuse instead of copying to each test

Reply:
rewritten task 2 as POM